flags/1x1/us.svg

```diff
@@ -1,14 +1,9 @@
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" id="flag-icons-us" viewBox="0 0 512 512" fill="#fff">
-  <path id="a" fill="#fff" d="M32 12l4 11h11l-9.1 6.7 3.5 10.7-9.4-6.7-8.7 6.7 3.6-10.7-9.5-6.7h11.8z"/>
-  <pattern id="p" patternUnits="userSpaceOnUse" width="65" height="55">
-    <use xlink:href="#a"/>
-  </pattern>
-  <pattern id="l" patternUnits="userSpaceOnUse" width="512" height="78.76">
-    <line stroke="#bd3d44" x2="512" x1="0" stroke-width="42" y1="18" y2="18"/>
-  </pattern>
-  <path d="M0 0h512v512H0"/>
-  <path fill="url(#l)" d="M0 0h512v512H0"/>
+<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" viewBox="0 0 512 512">
```

Comment:
Is xml:space requried? Please re-add the ID as well :)

Reply:
Note to my future self: stop opening pull requests early in the morning :)
I just corrected it!